config/routes.rb

```diff
@@ -16,7 +16,13 @@
           resources :sources
         end
       end
-      resources :events, only: [:create]
+      resources :events, only: [:create], controller: '/client_api/v1/events'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.